runtime/i386.S

```diff
@@ -113,15 +113,13 @@ G(caml_system__code_begin):
 
 FUNCTION(caml_call_gc)
         CFI_STARTPROC
+LBL(caml_call_gc):
     /* Record lowest stack address and return address */
-        pushl   %ebx; CFI_ADJUST(4)
         movl    G(Caml_state), %ebx
```

Comment:
I think you're right!

I've updated the other occurrence below (and done the same in i386nt.asm), but I'd prefer to leave this one as is. Removing it will make no perf difference (it's a cheap instruction on a codepath that does a lot of work), but makes the calling convention of caml_call_gc weirder.